test/IntegrationTests/GraphQLByteCodeTests.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="GraphQLByteCodeTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+public class GraphQLByteCodeTests : GraphQLTests
+{
+    public GraphQLByteCodeTests(ITestOutputHelper output)
+    : base("GraphQL", output)
+    {
+    }
+
+    public override bool SupportsExceptions => true;
+
+    public override string InstrumentationScope => "OpenTelemetry.AutoInstrumentation.GraphQL";
+
+    [Theory]
+    [Trait("Category", "EndToEnd")]
+    [MemberData(nameof(GetData), new object[] { false })]
```

Comment:
The `MemberData` here is pretty opaque, a comment or a constant to convey the meaning of the boolean would be helpful.

Reply:
Same for the "*.NativeTests.cs"

Reply:
@pjanotti I was also concerned about this. Either a new enum or 

```csharp
new object[] { /* isNativelyInstrumented */ false }
```

any preferences?